bundle/manifests/patterns-operator.clusterserviceversion.yaml

```diff
@@ -295,10 +278,19 @@ spec:
                     memory: 64Mi
                 securityContext:
                   allowPrivilegeEscalation: false
+                volumeMounts:
+                - mountPath: /gitops-config/
+                  name: gitops-config
+                  readOnly: true
               securityContext:
                 runAsNonRoot: true
               serviceAccountName: patterns-operator-controller-manager
               terminationGracePeriodSeconds: 10
+              volumes:
+              - configMap:
+                  name: patterns-operator-gitops-config
```

Comment:
what's the failure mode if the config map doesn't exist?
I don't see one being created by default

Reply:
It's optional at this point to make sure it's backwards compatible.